### DISCUSSION TITLE
简约样式 模糊

### DISCUSSION THREAD

**Discussion by GongYanYu** [xiaoshuapp/effector#16] [General]
body{
--card-background-color:#ffffff90;
  /* 隐藏*/
  .flow .flow-header,
  #banner,
  .toolbar {
    display: none !important;
  }

  .flow {
	z-index:0;
    margin-top: 6% !important;
  }

  .shu-card{
    backdrop-filter: blur(10px);
  }
}

![image](https://github.com/xiaoshuapp/effector/assets/43344004/59fce1d2-1773-41c4-bd66-42bf70c4e057)

**Comment by liyinred** [comment]
6666
